Replicator/DBAccess.cc

```diff
@@ -99,6 +99,11 @@ namespace litecore { namespace repl {
     }
 
 
+    const UseCollection DBAccess::useCollection(C4Collection* coll) const {
```

Comment:
`const UseCollection` seems meaningless.  What is it adding?  This is not a pointer or reference

Reply:
Only to make the code more fluent in the same spirit of useLocked.
Compare.   useLocked([](db) { collection->xyz() });
with            useCollection(collection)->xyz()
saving a lambda